README.md

```diff
@@ -29,7 +29,11 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 Write your code here...
 
 ```ruby
-# code here
+def offer_rose person
+  puts "Would you take this rose, #{person.to_s}, in exchange for giving an old beggar woman shelter from the bitter cold?"
+end
+
+offer_rose 'young prince'
 ```
 
 ### Question 2
```

Comment:
Should just be `#{person}`. The purpose of the template literal is to interpolate the code to a string, no need for `to_s`